api/catalog/api/utils/photon.py

```diff
@@ -58,35 +50,40 @@ def get(
         # do not serve over HTTP and do not have a redirect)
         params["ssl"] = "true"
 
-    # Photon excludes the protocol so we need to reconstruct the url + port + path
+    return params, parsed_image_url
+
+
+def get(
+    image_url: str,
+    accept_header: str = "image/*",
+    is_full_size: bool = False,
+    is_compressed: bool = True,
+) -> HttpResponse:
+    logger = parent_logger.getChild("get")
+    # Photon options documented here:
+    # https://developer.wordpress.com/docs/photon/api/
```

Comment:
nit: this comment should probably be in the `_get_photon_params` function instead.